src/reducers/browse.js

```diff
@@ -92,7 +94,7 @@ function browse(state = initialBrowseState, action) {
       return loop(
         state.set('isFetching', true),
         Cmd.run(fetchSubjects, {
-          args: [action.schoolId],
+          args: [action.currentTerm.get('id'), action.schoolId],
```

Comment:
We don't need the whole current term object. The enhancer should just provide the current term id.

Reply:
Done